extensions/podman/src/extension.ts

```diff
@@ -1052,6 +1051,7 @@ export function registerOnboardingRemoveUnsupportedMachinesCommand(): extensionA
         return !machine.json.GvProxy;
       });
 
+      console.log('invalid machines are', invalidMachines);
```

Comment:
to remove ? or use `console.(err | warn)` ?

Reply:
removed